switchover/dbsync/status.go

```diff
@@ -97,7 +97,7 @@ func (s *Sync) status(ctx context.Context) (string, error) {
 	buf.WriteString(table.String() + "\n\n")
 
 	if len(nodes) == 0 {
-		issues = append(issues, "No nodes detected.")
+		issues = append(issues, "No nodes detected.(Have you set `db-url-next` for running GoAlert instances?)")
```

Comment:
Maybe this one could be ` (make sure GoAlert is running)`

And for the `Non-switchover GoAlert connection` error, we can add ` (try setting --db-url-next)`

The other hints are super short & lower case, also don't forget the space before `(`